doc/source/conf.py

```diff
@@ -135,19 +119,18 @@
 # The master toctree document.
 master_doc = "index"
 
-suppress_warnings = ["autoapi.python_import_resolution", "config.cache"]
-#autoapi_python_class_content = "both"
-autoapi_member_order = "alphabetical"
-
 # TODO: remove ignore links after public release
 linkcheck_ignore = [
     "https://github.com/ansys/scade-apitools",
     "https://github.com/ansys/scade-apitools/actions/workflows/ci_cd.yml",
     "https://pypi.org/project/ansys-scade-apitools",
     # The link below takes a long time to check
     "https://www.ansys.com/products/embedded-software/ansys-scade-suite",
-    "https://www.ansys.com/*"
+    "https://www.ansys.com/*",
 ]
 
+if switcher_version != 'dev':
+    linkcheck_ignore.append(f'https://github.com/ansys/scade-apitools/releases/tag/v{__version__}')
```

Comment:
This will cause issues since your documentation builds before your project deploys to GitHub.

Reply:
Really? I don't think so, right?

They are ignoring the link check to a release that doesn't exist __yet__ but will exist once the release goes public. We are handling it this way in other repos too

Reply:
Oh! That's right. I misunderstood this logic.